ncutixavier/ncutixavier.github.io
# #JQTrfmoh Saving contact form into firestore

### **What about this PR?**
This PR is about to review and test the code of sending contact form values (name, email and message) to the firebase.

### **Description of Task to be completed**
It has the following items:
1. Page Title (Optional)
2. Validated Contact form
3. Popup message after sending successful
4. log the document from the collection of firebase (Optional)
5. Reload the page and check your console

### **How should be manually tested?**
1. Clone the repo
2. Open the folder "Personal-Website" after cloning or downloading
3. Open file "index.html" and **click on contact link in navigation bar** by using any browser you have like chrome, Firefox, ...
4. Fill and submit the contact form
5. Reload the page and check your logs

### **Relevant pivot tracker*
#JQTrfmoh
